Gemfile

```diff
@@ -27,7 +25,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
   gem 'rspec-rails', '~> 3.4'
-  gem 'saml_idp', '~> 0.4.0'
+  gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', branch: 'master'
```

Comment:
It's actually a bundler-audit offense. We must explicitly use HTTPS URLs for securitay.